Website/src/rendering/src/stories/Sessions/SessionInformation.stories.tsx

```diff
@@ -109,5 +109,8 @@ Default.args = {
     Date: { value: '2021-07-29T06:00:00Z' },
     Image: { value: { src: '/assets/img/shop/man-biker.jpg' } },
     Speakers: [speaker1, speaker2],
+    Rooms: [{ fields: { Name: { value: 'Room 1' } } }, { fields: { Name: { value: 'Room 2' } } }],
```

Comment:
Could these sections be better formatted in all stories files? I feel this is hard to read.